v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerSimpleIT.java

```diff
@@ -248,4 +464,288 @@ private void assertMovieTableContents() {
     Assert.assertEquals(123.098, numericVals.get(0).getBigDecimal(0).doubleValue(), 0.001);
     Assert.assertEquals(931.512, numericVals.get(1).getBigDecimal(0).doubleValue(), 0.001);
   }
+
+  private void assertAllDatatypeColumnsTableBackfillContents() {
+    List<Map<String, Object>> events = new ArrayList<>();
+
+    Map<String, Object> row = new HashMap<>();
+    row.put("varchar_column", "value1");
+    row.put("tinyint_column", "10");
+    row.put("text_column", "dGV4dF9kYXRhXzEK");
+    row.put("date_column", "2024-02-08T00:00:00.000Z");
+    row.put("smallint_column", "50");
+    row.put("mediumint_column", "1000");
+    row.put("int_column", "50000");
+    row.put("bigint_column", "987654321");
+    row.put("float_column", "45.67");
```

Comment:
Why are all the column values strings?

Reply:
Because the datatype mapping has been changed to string for all the columns to capture the case of datatype conversions supported via HB